src/purify.js

```diff
@@ -1202,10 +1204,6 @@ function createDOMPurify(window = getGlobal()) {
    * @param  {Node} currentNode to sanitize
    */
   const _sanitizeAttributes = function (currentNode) {
-    let attr = null;
-    let value = null;
-    let lcName = null;
-    let l = null;
```

Comment:
In my opinion, moving the variables inside the "while" block would make the code more intentional.